canonical/ubuntu.com
# WD-11740 Refreshed  /security/certifications

## Done

- Refreshed /security/certifications

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- [Demo](https://ubuntu-com-13993.demos.haus/security/certifications)
- [Copydoc](https://docs.google.com/document/d/13RcVvHBAoywfk3o5Dy-16F3KrMytL9-2v6wWbW444Ko/edit)
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
   

## Issue / Card

Fixes # [WD-11740](https://warthogs.atlassian.net/browse/WD-11740)

## Screenshots



## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-11740]: https://warthogs.atlassian.net/browse/WD-11740?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
